### PR TITLE
[skip ci] new questions in faq

### PR DESCRIPTION
**Pull request by salsakran**
Additional questions that keep filling my inbox.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -55,6 +55,16 @@ That said, we will build out additional database connectors as we are able to. W
 
 We welcome community contributions of database connectors. If you're able to help, we have [open issues](https://github.com/metabase/metabase/labels/Database%20Support) for a number of databases. We'd greatly appreciate your help!
 
+### Can Metabase connect to Google Analytics, Salesforce, etc?
+
+No. Metabase is a fast and easy way for you to access and share information you have in a database. We do not currently offer a way to connect to third party APIs or services directly. What people do instead in these situations is download data from these services into a database they control and then use Metabase to access that database directly. This can be done either by writing code or more commonly using a third party service. There are a large number of these service, and you can ask other users and discuss pro's and cons at our [user forum](https://discourse.metabase.com)
```
Comment (camsaul) [inline docs/faq.md]: ~~fast and easy~~
fast and steasy

Comment (mazameli) [inline docs/faq.md]: > There are a large number of these service, and you can ask other users and discuss pro's and cons at our…

There are a large number of these **services**, and you can ask other users and discuss **pros** and cons at our…

File: docs/faq.md [hunk 2]
```diff
@@ -55,6 +55,16 @@ That said, we will build out additional database connectors as we are able to. W
...
+
+One caveat is that if you are using Google Analytics Premium, one of the features is direct access to a BigQuery database with your personal Google Analytics data. In this situation, you can Metabase with that BigQuery dataset directly.
```
Comment (camsaul) [inline docs/faq.md]: ~~you can Metabase with that~~
you can use Metabase with that (unless you're trying to make Metabase as a verb a thing, which I support :+1:. "just Metabase it bruh!")

File: docs/faq.md [hunk 3]
```diff
@@ -55,6 +55,16 @@ That said, we will build out additional database connectors as we are able to. W
...
+
+### Can I upload data to Metabase?
+
+Not exactly. Metabase provides access to data you have in an existing database you control. We currently do not add or modify the information in your database. You should ask whoever controls the database you are accessing how to upload the data you're interested in accessing.
```
Comment (camsaul) [inline docs/faq.md]: ~~You should ask whoever~~
You should ask whomever

**Pull request merged**